cutest.rb

```diff
@@ -15,8 +15,7 @@ class Cutest < Formula
 
   depends_on "dpo/cutest/archdefs" => :build
   depends_on "dpo/cutest/sifdecode" => :build
-  depends_on "homebrew/versions/gcc43" => [:build, "enable-fortran"] if build.with? "matlab" # Mathworks only support gfortran 4.3.
-  depends_on :fortran
```

Comment:
We also need `:fortran` more generally, not just when `--with-matlab` is selected.